datadog_checks_dev/datadog_checks/dev/tooling/commands/create.py

```diff
@@ -108,6 +108,9 @@ def create(ctx, name, integration_type, location, non_interactive, quiet, dry_ru
     if os.path.exists(integration_dir):
         abort(f'Path `{integration_dir}` already exists!')
 
+    if integration_type == 'snmp_tile' and not name.startswith('snmp_'):
+        abort(f'Invalid snmp_tile name ({name}). snmp_tile integrations should start with `snmp_`')
```

Comment:
It seems that will force the to use `ddev create --type snmp_tile "snmp_<VENDOR>"`

I'm not sure that will lead to right naming for manifest.json display_name.

For example:

```
$ ddev create "snmp_cisco" --type snmp_tile
```

Will lead to `display_name: "snmp_cisco"`, but we probably want `display_name: Cisco` (or `display_name: SNMP Cisco`).

Note that the name passed to `ddev create <NAME>` should be the **display name** e.g. "My Vendor" (if I understanding correctly, that's the display name should only contain vendor name, no SNMP prefix)


We can possibly only pass the vendor name as `ddev create "My Vendor"` and then prefix with snmp when needed in template like: "snmp_my_vendor"

Reply:
I admit I tried doing it as you recommend, but couldn't find a way to create a folder using `snmp_<name>` in less than one day. Will go back to this solution, must dig deeper and longer into code.

Reply:
and found!